index.html

```diff
@@ -10270,6 +10270,12 @@ <h3><dfn>Perform Actions</dfn></h3>
 variables</var> and <var>parameters</var> are:
 
 <ol class="algorithm">
+  <li><p>If <var>session</var>&apos;s <a>current browsing context</a>
+  is <a>no longer open</a>, return <a>error</a> with <a>error code</a>
+  <a>no such window</a>.
+
+  <li><p><a>Try</a> to <a>handle any user prompts</a> with <var>session</var>.
+
```

Comment:
Moving this up as well given that there is no need to extract / process an action sequence when we have to fail anyway.